packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -212,7 +212,7 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
 
   @include mdc-theme-dark(".mdc-textfield", true) {
     &:not(.mdc-textfield--focused) {
-      &::after {
+      &.mdc-textfield--upgraded::after {
```

Comment:
we need both non-upgraded and upgraded after pseudo elements here. Both of these should read like so:

```
&::after,
&.mdc-textfield--upgraded::after {
```